pkgs/by-name/ol/ollama/cuda-path.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/llama/Makefile b/llama/Makefile
+index 47a87a75..8b71c999 100644
+--- a/llama/Makefile
++++ b/llama/Makefile
+@@ -15,7 +15,7 @@ else ifeq ($(OS),linux)
+ 	HIP_LIB_DIR := $(shell ls -d $(HIP_PATH)/lib 2>/dev/null)
+ 	CUDA_PATH?=/usr/local/cuda
+ 	CUDA_11:=$(shell ls -d $(CUDA_PATH)-11 2>/dev/null)
+-	CUDA_12:=$(shell ls -d $(CUDA_PATH)-12 2>/dev/null)
++	CUDA_12:=$(shell ls -d $(CUDA_PATH) 2>/dev/null)
+ endif
+ 
+ ifeq ($(OLLAMA_SKIP_CUDA_GENERATE),)
+diff --git a/llama/make/Makefile.cuda_v12 b/llama/make/Makefile.cuda_v12
+index 2418ef00..d27e5a84 100644
+--- a/llama/make/Makefile.cuda_v12
++++ b/llama/make/Makefile.cuda_v12
+@@ -5,7 +5,7 @@ include make/common-defs.make
+ 
+ GPU_RUNNER_VARIANT := _v12
+ GPU_PATH_ROOT_WIN=$(shell ls -d $(dir $(shell cygpath -m -s "$(CUDA_PATH)\.."))/v12.? 2>/dev/null)
+-GPU_PATH_ROOT_LINUX=$(shell ls -d $(CUDA_PATH)-12 2>/dev/null)
++GPU_PATH_ROOT_LINUX=$(shell ls -d $(CUDA_PATH) 2>/dev/null)
+ CUDA_ARCHITECTURES?=60;61;62;70;72;75;80;86;87;89;90;90a
+ 
```

Comment:
I've temporarily hardcoded it to CUDA 12 for verification purposes, and it works as expected. but I'm not sure of the best approach to support both CUDA 11 and 12. Any suggestions?